src/lib/components/statusicon/StatusIcon.component.tsx

```diff
@@ -0,0 +1,34 @@
+import { Icon, IconName } from '../icon/Icon.component';
+
+import {
+  STATUS_WARNING,
+  STATUS_CRITICAL,
+  STATUS_HEALTH,
+} from '../tablev2/TableUtils';
```

Comment:
Patrick move these variables in constants.ts, we should use them instead of importing them from table